src/ngraph/codegen/compiler.cpp

```diff
@@ -441,6 +473,67 @@ void codegen::CompilerCore::configure_search_path()
     {
         add_header_search_path("/usr/include");
     }
+
+#elif defined(_WIN32)
+    // Add base toolchain-supplied header paths
+    // We would use the MSVC toolchain definition in clang/lib/Driver/ToolChains/MSVC.h (copied to toolchains/MSVC.h)
+    // Unfortunately, the toolchain contains methods to get paths to libraries, not headers and some magic string
+    // manipulation is required.
+
+    auto& TO = m_compiler->getInvocation().getTargetOpts();
+    std::string Path = GetExecutablePath(m_source_name.c_str());
+
+    using clang::driver::toolchains::MSVCToolChain;
+    using clang::driver::Driver;
+
+    Driver TheDriver(Path, TO.Triple, m_compiler->getDiagnostics());
+    llvm::opt::InputArgList Args;
+    Triple TT(TO.Triple);
+
+    MSVCToolChain MSVC(TheDriver, TT, Args);
+
+    add_header_search_path(MSVC.getSubDirectoryPath(MSVCToolChain::SubDirectoryType::Include));
+
+    if (MSVC.useUniversalCRT()) {
```

Comment:
This file needs to conform to coding standard. If you  run `make style-apply` it will apply correct brace style, etc.

Reply:
Done.